src/js/common/schemaform/LoadingPage.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { LOAD_STATUSES } from './save-load-actions';
+import LoadingIndicator from '../components/LoadingIndicator';
+import SignInLink from '../components/SignInLink';
+import ProgressButton from '../components/form-elements/ProgressButton';
+
+
+// TODO: Determine whether I can connect() and withRouter() this component and still
+//  be alright for testing; the contamination in FormApp is getting silly.
+class LoadingPage extends React.Component {
```

Comment:
Like we talked about, this should probably be `LoadingScreen` or `SaveInProgressLoading` or something.